yellowbrick/features/pca.py

```diff
@@ -207,6 +214,10 @@ def finalize(self, **kwargs):
         self.ax.set_ylabel('Principal Component 2')
         if self.proj_dim == 3:
             self.ax.set_zlabel('Principal Component 3')
+        if self.legend_labels is not None:
+              manual_legend(
+              ax, self.legend_labels, self.color, frameon=True, loc='best'
```

Comment:
Looks like this is where the tests are failing; this should be `self.ax` instead of `ax`, right?